packages/graph/hash_graph/lib/graph/src/api/rest/json_schemas/update_data_type.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
These are literally just copy pastes of their non `update_` equivalents, with "$id" removed.

This is obviously less than ideal but I don't know how we want to address it right now without a tonne of overhead, and we might not be serving these meta-schemas statically for long anyway.